composer.json

```diff
@@ -27,6 +28,7 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "provide": {
+        "ext-mongo": "*",
```

Comment:
An idea: 
Using the same config as: https://github.com/doctrine/mongodb-odm for php7
But I don't know travis to understand the config:
https://github.com/doctrine/mongodb-odm/blob/master/.travis.yml